build.sh

```diff
@@ -70,8 +68,11 @@ qmake ../mupen64plus-gui.pro
 make -j4
 if [[ $UNAME == *"MINGW"* ]]; then
   cp $base_dir/mupen64plus-gui/build/release/mupen64plus-gui.exe $install_dir
+elif [[ $UNAME == *"Darwin"* ]]; then
+  /usr/local/Cellar/qt/5.15.1/bin/macdeployqt $base_dir/mupen64plus-gui/build/mupen64plus-gui.app
```

Comment:
It would be better not to hard code the path here, but I'm still figuring out how to get the path dynamicall from Brew.